diff -urN playonlinux/python/mainwindow.py playonlinux-patched/python/mainwindow.py
--- playonlinux/python/mainwindow.py	2014-09-08 07:43:37.000000000 +1100
+++ playonlinux-patched/python/mainwindow.py	2014-10-07 17:33:38.979620899 +1100
@@ -90,8 +90,6 @@
             self.updating = False
             if(playonlinux.VersionLower(os.environ["VERSION"],self.WebVersion)):
                 self.sendToStatusBar(_('An updated version of {0} is available').format(os.environ["APPLICATION_TITLE"])+" ("+self.WebVersion+")",False)
-                if(os.environ["DEBIAN_PACKAGE"] == "FALSE"):
-                    self.sendAlert(_('An updated version of {0} is available').format(os.environ["APPLICATION_TITLE"])+" ("+self.WebVersion+")")
                 os.environ["POL_UPTODATE"] = "FALSE"
             else:
                 self.Show = False
