diff -rupN playonlinux.old/etc/PlayOnLinux.desktop playonlinux/etc/PlayOnLinux.desktop
--- playonlinux.old/etc/PlayOnLinux.desktop	2013-03-31 23:12:23.000000000 +0200
+++ playonlinux/etc/PlayOnLinux.desktop	2013-05-06 10:28:02.415023534 +0200
@@ -2,8 +2,8 @@
 Version=1.0
 Encoding=UTF-8
 Name=PlayOnLinux
-Comment=PlayOnLinux
-Type=Application %F
+Comment=An manager for installing programs and games in Wine 
+Type=Application
 Exec=playonlinux
 Icon=/usr/share/playonlinux/etc/playonlinux.png
-Categories=Application;Game
+Categories=Game;
